ansible_rulebook/engine.py

```diff
@@ -462,7 +462,7 @@ def __init__(self):
         self.actions = OrderedDict()
         self.result = None
         self.stopped = True
-        self.delay = int(os.environ.get("EDA_RUN_PLAYBOOK_MAX_DELAY", 0))
+        self.delay = float(os.environ.get("EDA_RUN_PLAYBOOK_MAX_DELAY", 0.1))
```

Comment:
As I have commented in private conversations, IMO 0.1 is too high, I would put at least 0.01. Otherwise LGTM